scipy/spatial/distance.py

```diff
@@ -508,9 +512,11 @@ def minkowski(u, v, p=2, w=None):
         w = _validate_weights(w)
         if p == 1:
             root_w = w
-        if p == 2:
+        elif p == 2:
```

Comment:
A harmless bug. For `p=1` this previously would calculate `root_w = w ** 1` due to this not being an `elif`.